src/test/java/rv/atqc/gttesting/possibility_change_map_type/PossibilityChangeMapTypeChromePC.java

```diff
@@ -111,4 +111,15 @@ public void checkExistenceSatelliteItem() {
 						+ " hidden-xs map-menu-buttons-wrapper']//a[@id='satellite']")));
 	    Assert.assertTrue(satelliteItem.isEnabled());
 	}
+	
+	@Test
+	public void checkPossibilitySetStreetsMapType() {
+		WebElement streetsItem = wait.until(ExpectedConditions
+				.elementToBeClickable(By.xpath("//div[@class='col-lg-5 col-md-5 hidden-sm"
+						+ " hidden-xs map-menu-buttons-wrapper']//a[@id='streets']")));
+	    streetsItem.click();
```

Comment:
WebElement streetsItem = wait.until(ExpectedConditions
 +				.elementToBeClickable(By.xpath("//div[@class='col-lg-5 col-md-5 hidden-sm"
 +						+ " hidden-xs map-menu-buttons-wrapper']//a[@id='streets']")));
 +	    streetsItem.click();
Maybe this can be wrapped in one method:)